src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  deleteCompleted: () => void,
```

Comment:
proper naming is everything, here you go
```
const handleSomething = () => ...;
...

<Component onSomething={handleSomething} />
...

interface Props {
  onSomething: () => void;
}
```

Reply:
@jstmpelowycz Thanks for the advice, having problems with naming. I did as you proposed, but not sure. Please check it out.